packages/iproute/iproute.spec

```diff
@@ -45,12 +45,12 @@ export PKG_CONFIG_PATH='%{_cross_pkgconfigdir}' \\\
 %set_env
 %make_install
 
-install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
-mv %{buildroot}%{_cross_sysconfdir}/iproute2 %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
+install -d %{buildroot}%{_cross_factorydir}%{_cross_datadir}
+mv %{buildroot}%{_cross_datadir}/iproute2 %{buildroot}%{_cross_factorydir}%{_cross_datadir}
```

Comment:
This can't work correctly - `/usr/share` is read-only, so `systemd-tmpfiles` is probably complaining about this.

But the good news is that if `ip` can read from `/usr/share`, then we don't need any of this indirection via tmpfiles, and can just include those files directly in the read-only image.

Reply:
Thanks for calling this out, you are correct!